### PR TITLE
handle cv2 version greater than 3.0.0

### PR DESCRIPTION
**Pull request by Jambon1510**
Suppose to be able to handle cv2 version greater than 3.0.0
Only been tested with cv2 4.0.0.21

Related to #36

### CODE REVIEW

File: simpleocr/opencv_utils.py [hunk 1]
```diff
@@ -127,8 +128,8 @@ def get_opencv_version():
     Return the OpenCV version by checking cv2.__version__
     :return: int, 2 or 3
     """
-    if cv2.__version__.startswith("3"):
-        return 3
+    if version.parse(cv2.__version__) >= version.parse("3.0.0"):
```
Comment (goncalopp) [inline simpleocr/opencv_utils.py]: As much as I like the additional safety of parsing, I'm not sure it makes sense to introduce and extra dependency for a single line.
I think we can replace the whole function with `return int(cv2.__version__.split(".")[0])`, that's also what `imutils` is doing, so hopefully it almost always works
https://github.com/jrosebr1/imutils/blob/master/imutils/convenience.py#L216

**Comment by goncalopp** [comment]
By the way - you can always take a look at the status of the build in
https://travis-ci.org/goncalopp/simple-ocr-opencv

In [the build for this PR](https://travis-ci.org/goncalopp/simple-ocr-opencv/builds/506313800) case, the build fails, because you didn't include `packaging` in the `deps` section of `setup.py`. But let's remove the dependency instead :)

File: simpleocr/opencv_utils.py (hunk 1 above)
Comment (Jambon1510) [reply simpleocr/opencv_utils.py]: oh yep better indeed!

**Comment by Jambon1510** [comment]
oh yep, I thought PyCharm would have suggested me this.
I need to check to automatically doing it next time I am adding a new depedency.
Thx Goncalopp

**Comment by Jambon1510** [comment]
Committed and pushed
I realized that I forgot to do a special commit for segmentation. After some tests on cv2 4, it appears that I have the below message which I fixed by rollbacking  to the master on this particular line

`ValueError: not enough values to unpack (expected 3, got 2)`

**Comment by goncalopp** [comment]
Perfect! Travis is passing, so I'll merge this in
I'm surprised that the API for `findContours` in cv 4 is closer to version 2 - but we can always revisit this as needed
Thank you for contributing!

**Pull request merged**